diff --git a/infra/conf/transport_security.go b/infra/conf/transport_security.go
index b5cd7d2..d3f4ee6 100644
--- a/infra/conf/transport_security.go
+++ b/infra/conf/transport_security.go
@@ -358,9 +358,7 @@ func (c *TLSConfig) Build() (proto.Message, error) {
 	config.RejectUnknownSni = c.RejectUnknownSNI
 	config.MasterKeyLog = c.MasterKeyLog
 
-	if c.AllowInsecure {
-		return nil, errors.PrintRemovedFeatureError(`"allowInsecure"`, `"pinnedPeerCertSha256"(pcs) and "verifyPeerCertByName"(vcn)`)
-	}
+	config.AllowInsecure = c.AllowInsecure
 	if c.PinnedPeerCertSha256 != "" {
 		for v := range strings.SplitSeq(c.PinnedPeerCertSha256, ",") {
 			v = strings.TrimSpace(v)
diff --git a/transport/internet/tls/config.go b/transport/internet/tls/config.go
index f33e829..cbdb4a4 100644
--- a/transport/internet/tls/config.go
+++ b/transport/internet/tls/config.go
@@ -384,6 +384,7 @@ func (c *Config) GetTLSConfig(opts ...Option) *tls.Config {
 		PinnedPeerCertSha256: c.PinnedPeerCertSha256,
 	}
 	config := &tls.Config{
+		InsecureSkipVerify:     c.AllowInsecure,
 		Rand:                   randCarrier,
 		ClientSessionCache:     globalSessionCache,
 		RootCAs:                root,
diff --git a/transport/internet/tls/config.proto b/transport/internet/tls/config.proto
index a05cc04..a3ed860 100644
--- a/transport/internet/tls/config.proto
+++ b/transport/internet/tls/config.proto
@@ -38,7 +38,7 @@ message Certificate {
 }
 
 message Config {
-  // Number 1 was assigned and used by an legacy option.
+  bool allow_insecure = 1;
 
   // List of certificates to be served on server.
   repeated Certificate certificate = 2;
