diff --git a/app/src/main/java/com/oxygenupdater/OxygenUpdater.kt b/app/src/main/java/com/oxygenupdater/OxygenUpdater.kt
index 9b1c45a..d38c4a7 100644
--- a/app/src/main/java/com/oxygenupdater/OxygenUpdater.kt
+++ b/app/src/main/java/com/oxygenupdater/OxygenUpdater.kt
@@ -173,7 +173,7 @@ class OxygenUpdater : Application(), Configuration.Provider, SingletonImageLoade
     fun setupCrashReporting() {
         analytics.setUserProperty("device_name", SystemVersionProperties.deviceProductName)
 
-        val shouldShareLogs = sharedPreferences[KeyShareAnalyticsAndLogs, true]
+        val shouldShareLogs = false
         // Sync analytics collection to user's preference
         analytics.setAnalyticsCollectionEnabled(shouldShareLogs)
         // Sync crashlytics collection to user's preference, but only if we're on a release build
diff --git a/app/src/main/java/com/oxygenupdater/activities/MainActivity.kt b/app/src/main/java/com/oxygenupdater/activities/MainActivity.kt
index bb62227..23c2716 100644
--- a/app/src/main/java/com/oxygenupdater/activities/MainActivity.kt
+++ b/app/src/main/java/com/oxygenupdater/activities/MainActivity.kt
@@ -1112,6 +1112,7 @@ class MainActivity : AppCompatActivity() {
     private val mobileAdsInitDone = AtomicBoolean(false)
     private fun setupMobileAds() {
         if (mobileAdsInitDone.get()) return else mobileAdsInitDone.set(true)
+        if (true) return // patch: skip ad SDK init
 
         // Init Google Mobile Ads SDK on a background thread
         CoroutineScope(Dispatchers.IO).launch {
diff --git a/app/src/main/java/com/oxygenupdater/ui/common/BannerAd.kt b/app/src/main/java/com/oxygenupdater/ui/common/BannerAd.kt
index ced7fad..73e0946 100644
--- a/app/src/main/java/com/oxygenupdater/ui/common/BannerAd.kt
+++ b/app/src/main/java/com/oxygenupdater/ui/common/BannerAd.kt
@@ -49,86 +49,7 @@ fun ColumnScope.BannerAd(
     navType: NavType,
     forceAdReload: Boolean,
     addNavBarPadding: Boolean,
-): Unit = if (LocalInspectionMode.current) {
-    Text("AdView", Modifier.align(Alignment.CenterHorizontally))
-} else {
-    var loaded by rememberState(false)
-
-    val density = LocalDensity.current.density
-    val adSize = remember(navType) {
-        val adWidthPx = if (SDK_INT >= VERSION_CODES.R) {
-            val metrics = activity.windowManager.currentWindowMetrics
-            val insets = metrics.windowInsets.getInsets(WindowInsets.Type.systemBars())
-
-            metrics.bounds.width() - (insets.right + insets.left)
-        } else activity.resources.displayMetrics.widthPixels
-
-        /**
-         * Note: keep in sync with
-         * [androidx.compose.material3.tokens.NavigationRailCollapsedTokens.NarrowContainerWidth]
-         * */
-        val sideRailWidth = if (navType == NavType.SideRail) 80 + 1 else 0
-        val adWidth = (adWidthPx / density.fastCoerceAtLeast(1f)).toInt() - sideRailWidth
-
-        /**
-         * TODO(ads): [AdView.setAdSize] can only be called once per [AdView], which means we can't
-         *  update its size when [navType] changes. Figure out how to discard the old one and reload.
-         */
-        AdSize.getLargeAnchoredAdaptiveBannerAdSize(activity, adWidth)
-    }
-
-    val adView = remember {
-        AdView(activity).apply {
-            // Both of these can only be set once per AdView
-            adUnitId = BuildConfig.AD_BANNER_MAIN_ID
-            setAdSize(adSize).also { logDebug(TAG, "Requested size: $adSize") }
-
-            adListener = object : AdListener() {
-                override fun onAdFailedToLoad(error: LoadAdError) {
-                    loaded = false
-                    logDebug(TAG, "Banner ad failed to load: $error")
-                }
-
-                override fun onAdLoaded() {
-                    loaded = true
-                    val adapterResponseInfo = responseInfo?.loadedAdapterResponseInfo
-                    val adapterClassName = adapterResponseInfo?.adapterClassName
-                    val adSourceName = adapterResponseInfo?.adSourceName
-                    val latencyMillis = adapterResponseInfo?.latencyMillis
-                    logDebug(TAG, "$adapterClassName: LOADED $adSourceName $adSize ($latencyMillis ms)")
-                }
-            }
-        }
-    }
-
-    LaunchedEffect(forceAdReload) {
-        if (loaded || adView.isLoading) {
-            logDebug(TAG, "Skipping load, previous is pending (loaded=$loaded, adViewIsLoading: ${adView.isLoading})")
-            return@LaunchedEffect
-        }
-
-        logDebug(TAG, "Loading")
-        adView.loadAd(buildAdRequest())
-    }
-
-    AndroidView(
-        factory = { adView },
-        // We draw the activity edge-to-edge, so nav bar padding should be applied only if ad loaded
-        modifier = (if (addNavBarPadding && loaded) Modifier.navigationBarsPadding() else Modifier)
-            .align(Alignment.CenterHorizontally)
-            .wrapContentSize()
-    )
-
-    // Pause and resume the AdView when the lifecycle is paused and resumed
-    LifecycleResumeEffect(adView) {
-        adView.resume()
-        onPauseOrDispose { adView.pause() }
-    }
-
-    DisposableEffect(Unit) {
-        // Destroy the AdView to prevent memory leaks when the screen is disposed
-        onDispose { adView.destroy() }
-    }
+) {
 }
 
 /**
