diff --git a/app/src/main/java/com/oxygenupdater/OxygenUpdater.kt b/app/src/main/java/com/oxygenupdater/OxygenUpdater.kt
--- a/app/src/main/java/com/oxygenupdater/OxygenUpdater.kt
+++ b/app/src/main/java/com/oxygenupdater/OxygenUpdater.kt
@@ -173,7 +173,7 @@ class OxygenUpdater : Application(), Configuration.Provider, SingletonImageLoade
     fun setupCrashReporting() {
         analytics.setUserProperty("device_name", SystemVersionProperties.deviceProductName)

-        val shouldShareLogs = sharedPreferences[KeyShareAnalyticsAndLogs, true]
+        val shouldShareLogs = false
         // Sync analytics collection to user's preference
         analytics.setAnalyticsCollectionEnabled(shouldShareLogs)
         // Sync crashlytics collection to user's preference, but only if we're on a release build
